### PR TITLE
Fb asim 4132 global trends

### PR DESCRIPTION
**Pull request by arthursoprana**

### CODE REVIEW

File: docs/source/alfacase_definitions/CaseOutputDescription.txt
Comment (prusse-martin) [inline docs/source/alfacase_definitions/CaseOutputDescription.txt:32]: This is just a bookmark.

File: src/alfasim_sdk/_internal/alfacase/generate_case_description_docstring.py
Comment (prusse-martin) [inline src/alfasim_sdk/_internal/alfacase/generate_case_description_docstring.py:172]: This is just a bookmark.

**Comment by nicoddemus** [comment]
Also mind the commits, there's a "WIP" in there that should be cleaned up before merging.

File: HISTORY.rst
Comment (nicoddemus) [inline HISTORY.rst:28]:
```suggestion
* Add a new category ``gas standard volume`` from quantity ``standard volume``.
```

**Review by nicoddemus — APPROVED** [review]

File: tests/alfacase/test_generate_case_description_docstring.py
Comment (prusse-martin) [inline tests/alfacase/test_generate_case_description_docstring.py:17]: I don't want to block the PR merging for this.
Next time maybe do not commit this change =)

File: docs/source/alfacase_definitions/AnnulusDescription.txt
Comment (prusse-martin) [inline docs/source/alfacase_definitions/AnnulusDescription.txt:24]: I think we did add the escape at the end back. I can't find the code.

```
                    # Remove the last character if it's a backslash otherwise the line break will be avoided.
                    if line_content.endswith("\\"):
                        line_content = line_content[:-1]
```

OK, since the `# optional` moved the last escape is removed.

**Review by prusse-martin — APPROVED** [review]

File: docs/source/alfacase_definitions/AnnulusDescription.txt
Comment (oliveira-mauricio) [reply docs/source/alfacase_definitions/AnnulusDescription.txt:24]: Just to confirm, it is rendering correctly:

image

https://alfasim-sdk--158.org.readthedocs.build/en/158/alfacase/api_reference.html#alfasim_sdk.AnnulusDescription

Comment (oliveira-mauricio) [reply docs/source/alfacase_definitions/AnnulusDescription.txt:24]: Duplicities handled in another PR: https://github.com/ESSS/alfasim-sdk/pull/164

**Pull request merged**
